src/RocketSection/RocketList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Rocket from './Rocket';
+// import { getAllRocketsAsync } from '../reduxStore/rockets';
+
+function RocketList() {
+  // const dispatch = useDispatch();
+  const rocketTest = useSelector((state) => state.rocket);
+  // useEffect(() => {
+  //   dispatch(getAllRocketsAsync());
+  // }, [dispatch]);
```

Comment:
- [ ] Kindly remove the unused code to make your code clean and follow JavaScript best practices.